app.js

```diff
@@ -0,0 +1,71 @@
+var playerInput = prompt("Choose your weapon: rock, paper, or scissors. Type EXIT to exit the app.");
+
+// LOOP TO MAKE THE PROMPT REAPPEAR
+while (playerInput !="EXIT") {
+
+  // THE COMPUTER'S RANDOM MOVE
+  var computerMove = Math.random();
+  if (computerMove < 0.34){
+    computerMove = "rock";
+  }
+  else if (computerMove <= 0.67){
+    computerMove = "paper";
+  }
+  else {
+    computerMove = "scissors";
+  }
+
+// COMPARING THE TWO SCORES TO SEE WHO'S WON
+  function checkScore(playerInput, computerMove){
+
+    if (playerInput === computerMove){
+      alert("It's a draw!");
```

Comment:
playerScore and computerScore aren't defined in this one, hence why you get undefined outputted when you alert displayScore.